src/components/radio/radio_button.html

```diff
@@ -0,0 +1,22 @@
+<!-- TODO(jelbourn): render the radio on either side of the content -->
+<!-- TODO(mtlin): Evaluate trade-offs of using native radio vs. cost of additional bindings. -->
+
+<!-- The actual `radio` part of the control. -->
+<div class="md-radio-container">
+  <div class="md-radio-outer-circle"></div>
+  <div class="md-radio-inner-circle"></div>
+  <div class="md-ink-ripple"></div>
+</div>
+
+<input #input class="md-radio-input" type="radio"
+        [checked]="checked"
+        [disabled]="disabled"
+        [name]="name"
+        (change)="onInputChange($event)"
+        (focus)="onInputFocus()"
+        (blur)="onInputBlur()" />
+
+<!-- The label for radio control. -->
+<div class="md-radio-label">
```

Comment:
The text should label the input, either using a `<label>` element or with `aria-labelledby`